ops/cmd/check_chainlist/main.go

```diff
@@ -21,38 +21,40 @@ func mainErr() error {
 		return fmt.Errorf("failed to get working directory: %w", err)
 	}
 
-	chainCfg, err := manage.StagedChainConfig(wd)
+	stagedChainCfgs, err := manage.StagedChainConfigs(wd)
 	if errors.Is(err, manage.ErrNoStagedConfig) {
 		output.WriteOK("no staged chain config found, exiting")
 		return nil
 	}
 	if err != nil {
-		return fmt.Errorf("failed to get staged chain config: %w", err)
+		return fmt.Errorf("failed to get staged chain configs: %w", err)
 	}
 
-	output.WriteOK("validating uniqueness for chain ID %d and short name %s", chainCfg.ChainID, chainCfg.ShortName)
+	for _, chainCfg := range stagedChainCfgs {
+		output.WriteOK("validating uniqueness for chain ID %d and short name %s", chainCfg.ChainID, chainCfg.ShortName)
 
-	globalChainData, err := manage.FetchGlobalChainIDs()
-	if err != nil {
-		return fmt.Errorf("failed to fetch global chain IDs: %w", err)
-	}
-	output.WriteOK("fetched global chain IDs")
+		globalChainData, err := manage.FetchGlobalChainIDs()
+		if err != nil {
+			return fmt.Errorf("failed to fetch global chain IDs: %w", err)
+		}
+		output.WriteOK("fetched global chain IDs")
 
-	entry, ok := globalChainData[chainCfg.ChainID]
-	if !ok {
-		return fmt.Errorf("chain ID %d not found in global chain data", chainCfg.ChainID)
-	}
-	output.WriteOK("chain ID %d found in global chain data", chainCfg.ChainID)
+		entry, ok := globalChainData[chainCfg.ChainID]
+		if !ok {
+			return fmt.Errorf("chain ID %d not found in global chain data", chainCfg.ChainID)
```

Comment:
We could be a bit fancier here and collect all of the errors so we don't just fail as soon as there is one. Not sure it is worth it, though.